Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -135,6 +137,9 @@ open class MapView: UIView {
         } else {
             resolvedMapInitOptions = mapInitOptions
         }
+
+        self.pixelRatio = CGFloat(resolvedMapInitOptions.mapOptions.pixelRatio)
```

Comment:
What would be the best place to document that for externally connected display, e.g., when [CarPlay window is connected](https://developer.apple.com/documentation/carplay/cptemplateapplicationscenedelegate/3340482-templateapplicationscene?language=objc), developer has to provide `MapOptions.pixelRatio` of a connected screen? For example `CPWindow.screen.scale`.

Reply:
I'd say the Migration Guide. @knov thoughts about this?